### PR TITLE
Refactor max-bytes to be setup in prospector

### PR DESCRIPTION
**Pull request by ruflin**
Improve harvester closing logging

### CODE REVIEW

**Comment by ruflin** [comment]
@urso Can you review this one as MaxBytes was introduced by you. Behaviour should still be the same.

File: filebeat/crawler/prospector.go [hunk 1]
```diff
@@ -185,6 +185,10 @@ func (p *Prospector) setupHarvesterConfig() error {
 		return err
 	}
 
+	if config.MaxBytes == 0 {
+		config.MaxBytes = cfg.DefaultMaxBytes
+	}
```
Comment (urso) [inline filebeat/crawler/prospector.go]: reminds me:
- what if max_bytes < 0 ?
- do we want to impose limit if max_bytes == 0?

Comment (ruflin) [inline filebeat/crawler/prospector.go]: Should we make it here config.MaxBytes <= 0. We should also log what value was set (will add that). What would happen if we use uint as the variable and some uses -14 in the config?

Comment (ruflin) [inline filebeat/crawler/prospector.go]: @urso I updated it now with -1 and added logging

**Comment by urso** [comment]
LGTM

**Pull request merged**